skins/shared/color.scss

```diff
@@ -201,3 +201,9 @@ $color-vertical-tv-secondary: #008cce;
 // Venus article
 $color-article-navigation-inactive: if($is-dark-wiki, mix(#fff, $color-page, 30%), mix(#000, $color-page, 30%));
 $image-caption-border-color: if($is-dark-wiki, darken($color-text, 30%), lighten($color-text, 30%));
+
+//New site modernizations module background
```

Comment:
`New` doesn't look good. I'm sure there will come newer changes. And then 'newerer', 'newererst', 'new newerest'… :wink: You could use something non-relative, like a ticket name or date.